Classification/examples/Classification/CMakeLists.txt

```diff
@@ -29,6 +29,9 @@ endif()
 find_package( Boost OPTIONAL_COMPONENTS serialization iostreams )
 
 if( WIN32 )
+# to avoid a warning with old cmake 
+  set(_Boost_BZIP2_HEADERS             "boost/iostreams/filter/bzip2.hpp")
+  set(_Boost_ZLIB_HEADERS              "boost/iostreams/filter/zlib.hpp")
```

Comment:
Andreas, the same workaround is required also in the Polyhedron demo:
https://cgal.geometryfactory.com/CGAL/testsuite/CGAL-4.12-Ic-164/Polyhedron_Demo/TestReport_afabri_x64_Cygwin-Windows8_MSVC2013-Release-32bits.gz
```
-- Found the following Boost libraries:
--   serialization
--   iostreams
--   regex
CMake Warning at C:/Program Files/CMake/share/cmake-3.6/Modules/FindBoost.cmake:1459 (message):
  No header defined for zlib; skipping header check
Call Stack (most recent call first):
  Plugins/Classification/CMakeLists.txt:11 (find_package)


-- Boost version: 1.57.0
-- Found the following Boost libraries:
--   zlib
```